api/envoy/api/v2/cds.proto

```diff
@@ -385,6 +385,13 @@ message Cluster {
     // going to an individual locality if said locality is disproportionally affected by the
     // subset predicate.
     bool scale_locality_weight = 5;
+
+    // If true, when a fallback policy is configured and its corresponding subset fails to find
+    // a host this will cause any host to be selected instead.
+    //
+    // This is useful when using the default subset as the fallback policy, given the default
+    // subset might become empty which would cause requests to fail unless this option is enabled.
+    bool panic_mode_any = 6;
```

Comment:
Thoughts on naming this something more identifiable? Would be nice be able to clearly differentiate this from the panic mode used by the LB. maybe `subset_panic_mode`?

Reply:
Hmm, we are within the `LbSubsetConfig` message already, so not sure we need subset again. I actually like `panic_mode_any` because it a) sorta is like the other panic mode people know about and b) vaguely reference the likeliness with the ANY fallback policy... 

But, do pushback if you feel strongly about this :-)

Reply:
I don't feel strongly about it, it being nested is probably good for now. My concern was about confusing the two panic modes precisely because they are fairly similar, but we can just refer to this as subset panic mode elsewhere so I think it's fine